scripts/ci/check-destructive-migrations.js

```
@@ -0,0 +1,263 @@
+
+const { execFileSync } = require('node:child_process');
+const fs = require('node:fs');
+const os = require('node:os');
+const path = require('node:path');
+
+const DEFAULT_LABEL = 'allow-destructive-migration';
+const SCHEMA_PATH = 'prisma/schema.prisma';
+const MIGRATIONS_ROOT = 'prisma/migrations/';
+const MAX_STATEMENT_LENGTH = 220;
+const DESTRUCTIVE_PATTERNS = [
+  { kind: 'DROP TABLE', regex: /\bDROP\s+TABLE\b/i },
```

ESLint

[error] Replace `'DROP·TABLE'` with `"DROP·TABLE"`

(prettier/prettier)